cloudformation/tasking-manager.template.js

```diff
@@ -0,0 +1,201 @@
+    Description: 'Remote RDS URL',
+    Type: 'String'
+  }
+};
+
+const Conditions = {
+  UseASnapshot: cf.notEquals(cf.ref('DBSnapshot'), '')
+};
+
+const Resources = {
+  TaskingManagerASG: {
+    DependsOn: 'TaskingManagerLaunchConfiguration',
+    Type: 'AWS::AutoScaling::AutoScalingGroup',
+    Properties: {
+      AutoScalingGroupName: cf.stackName,
+      Cooldown: 300,
+      MinSize: 1,
+      DesiredCapacity: 1,
+      MaxSize: 5,
+      HealthCheckGracePeriod: 300,
+      LaunchConfigurationName: cf.ref('TaskingManagerLaunchConfiguration'),
+      LoadBalancerNames: [ cf.ref('TaskingManagerLoadBalancer') ],
+      HealthCheckType: 'EC2',
+      AvailabilityZones: cf.getAzs(cf.region)
+    }
+  },
+  TaskingManagerScaleUp: {
+      Type: "AWS::AutoScaling::ScalingPolicy",
+      Properties: {
+        AutoScalingGroupName: cf.ref('TaskingManagerASG'),
+        PolicyType: 'TargetTrackingScaling',
+        TargetTrackingConfiguration: {
+          TargetValue: 85,
+          PredefinedMetricSpecification: {
+            PredefinedMetricType: 'ASGAverageCPUUtilization'
+          }
+        },
+        Cooldown: 300
+      }
+  },
+  TaskingManagerLaunchConfiguration: {
+    Type: 'AWS::AutoScaling::LaunchConfiguration',
+      Properties: {
+        IamInstanceProfile: cf.ref('TaskingManagerEC2InstanceProfile'),
+        ImageId: 'ami-0e4372c1860d7426c',
+        InstanceType: 'm3.medium',
+        SecurityGroups: [cf.ref('RDSSecurityGroup')],
+        UserData: cf.userData([
+          '#!/bin/bash',
+          'set -x',
+          'export DEBIAN_FRONTEND=noninteractive',
+          'export LC_ALL="en_US.UTF-8"',
+          'export LC_CTYPE="en_US.UTF-8"',
+          'dpkg-reconfigure --frontend=noninteractive locales',
+          cf.sub('sudo apt-get update && sudo apt-get -y upgrade && sudo add-apt-repository ppa:jonathonf/python-3.6 -y && sudo apt-get update && sudo apt-get -y install python3.6 && sudo apt-get -y install python3.6-dev && sudo apt-get -y install python3.6-venv && sudo apt-get -y install curl && curl -sL https://deb.nodesource.com/setup_6.x > install-node6.sh && sudo chmod +x install-node6.sh && sudo ./install-node6.sh && sudo apt-get -y install nodejs && sudo npm install gulp -g && npm i browser-sync --save && sudo apt-get -y install postgresql-9.5 && sudo apt-get -y install libpq-dev && sudo apt-get -y install postgresql-server-dev-9.5 && sudo apt-get -y install libxml2 && sudo apt-get -y install libxml2-dev && sudo apt-get -y install libgeos-3.5.0 && sudo apt-get -y install libgeos-dev && sudo apt-get -y install libproj9 && sudo apt-get -y install libproj-dev && sudo apt-get -y install libgdal1-dev && sudo apt-get -y install libjson-c-dev && sudo apt-get -y install git && git clone --recursive https://github.com/hotosm/tasking-manager.git && cd tasking-manager/ && git reset --hard ${GitSha} && python3.6 -m venv ./venv && . ./venv/bin/activate && pip install --upgrade pip && pip install -r requirements.txt'),
```

Comment:
Is there a specific reason a special repository for the inclusion of Python 3.6 is added here. Wouldn't it be easier to run on the Python 3 version of Debian (which is 3.7 on the current stable version) and the TM runs well with it.